### DISCUSSION TITLE
Wrong type in v-for

### DISCUSSION THREAD

**Discussion by woldtwerk** [nuxt/nuxt#25861] [Questions] [answered]
Recently I encountered this weird problem. I thought quite some time about this, but can't come up with any reason.
Where does the true come from in the v-for?

![image](https://github.com/nuxt/nuxt/assets/32794633/a7c63554-e26b-4408-9657-4a283c96497c)

![image](https://github.com/nuxt/nuxt/assets/32794633/46d300aa-54d7-49a7-8174-0b1e4896090b)

**Comment by woldtwerk** [comment]
```vue
v-for="tag in tags.value"
``` 
works in the IDE, but of course bricks the app.

**Comment by TheAlexLichter** [comment, marked as answer]
See https://github.com/nuxt/nuxt/issues/25698

**Reply by woldtwerk** [reply]
Thanks!
my loop works again.
